docs/integration.md

```diff
@@ -0,0 +1,102 @@
+
+The endpoint accepts a `POST` request with
+
+#### Request Body
+
+| Variable Name | Type                          | Description         |
+| ------------- | ----------------------------- | ------------------- |
+| id            | string                        | event id            |
+| type          | enum: [EventType](#eventtype) |
+| data          | Object                        | any additional data |
+
+#### EventType
+
+| Value                    | Description                                                                 |
+| ------------------------ | --------------------------------------------------------------------------- |
+| IncomingPaymentCompleted | Incoming payment is complete and doesn't accept any incoming funds anymore. |
+| IncomingPaymentExpired   | Incoming payment is expired and doesn't accept any incoming funds anymore.  |
+| OutgoingPaymentCreated   | Outgoing payment was created.                                               |
+| OutgoingPaymentCompleted | Outgoing payment is complete.                                               |
+| OutgoingPaymentFailed    | Outgoing payment failed.                                                    |
+
+The `backend` package requires an environment variable called `WEBHOOK_URL` which MUST specify this endpoint.
+
+### Event Handlers
+
+#### `IncomingPaymentCompleted`
+
+An [Open Payments](./glossary#open-payments) Incoming Payment was completed, either manually or programmatically, i.e. it does not accept any incoming funds anymore. The Account Servicing Entity SHOULD withdraw any funds already received and deposit them into the payee's account.
+
+- Action: Withdraw liquidity
+
+#### `IncomingPaymentExpired`
+
+An [Open Payments](./glossary#open-payments) Incoming Payment has expired, i.e. it does not accept any incoming funds anymore. The Account Servicing Entity SHOULD withdraw any funds already received and deposit them into the payee's account.
+
+- Action: Withdraw liquidity
+
+#### `OutgoingPaymentCreated`
+
+An [Open Payments](./glossary#open-payments) Outgoing Payment has been created. It requires liquidity to be processed. The Account Servicing Entity SHOULD reserve the maximum requisite funds for the payment attempt on the payer's account.
+
+- Action: Deposit liquidity
+
+#### `OutgoingPaymentCompleted`
+
+An [Open Payments](./glossary#open-payments) Outgoing Payment was completed, i.e. it won't send any further funds. The Account Servicing Entity SHOULD withdraw any excess liquidity and deposit it into the payer's account.
+
+- Action: Withdraw liquidity
+
+#### `OutgoingPaymentFailed`
+
+An [Open Payments](./glossary#open-payments) Outgoing Payment completely of partially failed to send funds and won't re-try sending them. The Account Servicing Entity SHOULD withdraw all or any excess liquidity and deposit it into the payer's account.
```

Comment:
completely of -> completely or ?

Reply:
"An Open Payments Outgoing Payment failed to send all (or any) of the funds and won't re-try. The Account Servicing Entity SHOULD return any unsent to the payer's account."